packages/exoflex/src/components/RichRadio/RichRadioItem.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+
+import Text from '../Text';
+
+import useTheme from '../../helpers/useTheme';
+
+export type RichRadioItemProps = {
+  label: string;
+  selectedColor?: string;
+  selected?: boolean;
```

Comment:
I kinda forgot, do we use `selected` or `isSelected` in other components?

Reply:
We used `visible` and `checked` in other components.